test/pkg/gitlab/crd.go

```diff
@@ -12,7 +12,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func CreateCRD(ctx context.Context, projectinfo *gitlab.Project, run *params.Run, targetNS string, incomings *[]v1alpha1.Incoming) error {
+func CreateCRD(ctx context.Context, projectinfo *gitlab.Project, run *params.Run, targetNS string, incomings *[]v1alpha1.Incoming, settings *v1alpha1.Settings) error {
```

Comment:
```suggestion
func CreateCRD(ctx context.Context, projectinfo *gitlab.Project, run *params.Run, targetNS string, incomings *[]v1alpha1.Incoming, opts options.E2E) error {
```

@PuneetPunamiya can you please add settings in [E2E options](https://github.com/openshift-pipelines/pipelines-as-code/blob/3e661a939728ab9942bb45bbd343834854049db4/test/pkg/options/options.go#L3) and adding it as param in CreateCRD like it is done in Github's [CreateCRD](https://github.com/openshift-pipelines/pipelines-as-code/blob/3e661a939728ab9942bb45bbd343834854049db4/test/pkg/github/crd.go#L18)